### PR TITLE
Add support for foreman-puma-status command

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by evgeni** [comment]
can you also add changes for focal, now that we support it?

File: debian/bionic/foreman/control [hunk 1]
```diff
@@ -210,7 +210,7 @@ Package: foreman-service
 Architecture: all
 Section: web
 Priority: extra
-Depends: foreman (= ${binary:Version}), ${misc:Depends}
+Depends: foreman (= ${binary:Version}), puma-status, ${misc:Depends}
```
Comment (evgeni) [inline debian/bionic/foreman/control:213]: where is this package coming from? I don't see it in Debian or our repos?

Comment (ehelms) [reply debian/bionic/foreman/control:213]: Honestly, I don't know what determines when a package comes from one place or the other given we use bundle install for dependencies?

Comment (evgeni) [reply debian/bionic/foreman/control:213]: We bundle things that are in bundler.d, not from the Debian Depends.

**Comment by evgeni** [comment]
replaced by https://github.com/theforeman/foreman-packaging/pull/6567

**Review by DaviidFr — COMMENTED** [review]
Do we have to get some @gas

**Pull request closed without merging**